app/uk/gov/hmrc/economiccrimelevyreturns/connectors/EmailConnector.scala

```diff
@@ -53,4 +54,20 @@ class EmailConnector @Inject() (appConfig: AppConfig, httpClient: HttpClient)(im
         case Right(_) => ()
       }
 
+  def sendAmendReturnSubmittedEmail(to: String, parameters: AmendReturnSubmittedParameters)(implicit
+    hc: HeaderCarrier
+  ): Future[Unit] =
+    httpClient
+      .POST[AmendReturnSubmittedRequest, Either[UpstreamErrorResponse, HttpResponse]](
+        sendEmailUrl,
+        AmendReturnSubmittedRequest(
+          to = Seq(to),
+          templateId = AmendReturnTemplateId,
+          parameters = parameters
+        )
+      )
+      .map {
+        case Left(e)  => throw e
```

Comment:
You shouldn't need to throw this UpstreamErrorResponse. You can just return it and handle in the EmailService. Throwing exceptions is expensive.